frontend/src/app/main/ui/workspace/tokens/sets.cljs

```diff
@@ -224,7 +222,9 @@
      [:> icon-button*
       {:class (stl/css :set-item-group-collapse-button)
        :on-click on-collapse-click
+       :data-testid "tokens-set-group-collapse"
        :aria-label (tr "labels.collapse")
+       :title (tr "labels.collapse")
```

Comment:
title is not necessary, aria-label will be use as title inside component

Reply:
Ups I did not know that